moxy-compiler/src/main/java/moxy/compiler/viewstate/ViewInterfaceProcessor.kt

```diff
@@ -236,13 +237,42 @@ class ViewInterfaceProcessor(
         return modifiers.contains(Modifier.STATIC)
     }
 
-    private inline fun <T, R> List<T>.flatMapToSet(transform: (T) -> Iterable<R>): Set<R> {
-        return flatMapTo(LinkedHashSet(), transform)
-    }
-
     companion object {
         private const val OPTION_DEFAULT_STRATEGY = MvpCompiler.DEFAULT_MOXY_STRATEGY
         private val DEFAULT_STATE_STRATEGY: TypeElement =
             elementUtils.getTypeElement(AddToEndSingleStrategy::class.java.canonicalName)
     }
+
+    class StockViewMethod(
+            val targetInterfaceElement: DeclaredType,
+            val element: ExecutableElement,
+            val strategy: TypeElement?,
+            val tag: String
+    )
```

Comment:
`StockViewMethod` can be data class
`StockViewMethod` can be extracted into a separate file

Reply:
Y, he can, but not reason in hashCode and equals, they not used.
Its accessory class, i think he must be private internal class.